docs/qcqmc/full_workflow.ipynb

```diff
@@ -0,0 +1,363 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define an SCF Job\n",
+    "\n",
+    "We first setup a `PyscfHamiltonianParams` object which defines the SCF job we will run using [pyscf](https://github.com/pyscf/pyscf). Here we are simulation H4 in the (minimal) sto-3g basis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.qcqmc.hamiltonian import PyscfHamiltonianParams, HamiltonianData\n",
+    "\n",
+    "pyscf_params = PyscfHamiltonianParams(\n",
+    "    name=\"TEST_square_H4\",\n",
+    "    n_orb=4,\n",
+    "    n_elec=4,\n",
+    "    geometry=((\"H\", (0, 0, 0)), (\"H\", (0, 0, 1.23)), (\"H\", (1.23, 0, 0)), (\"H\", (1.23, 0, 1.23))),\n",
+    "    basis=\"sto3g\",\n",
+    "    multiplicity=1,\n",
+    "    charge=0,\n",
+    "    save_chkfile=True,\n",
+    "    overwrite_chk_file=True,\n",
+    ")\n",
+    "pyscf_hamiltonian = HamiltonianData.build_hamiltonian_from_pyscf(pyscf_params)\n",
+    "chk_path = pyscf_params.base_path.with_suffix(\".chk\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Build Perfect Pairing Wavefunction\n",
+    "\n",
+    "Next we build a perfect pairing Hamiltonian based upon our `pyscf_params`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from recirq.qcqmc.trial_wf import (\n",
+    "    PerfectPairingPlusTrialWavefunctionParams,\n",
+    "    TrialWavefunctionData\n",
+    ")\n",
+    "\n",
+    "pp_params = PerfectPairingPlusTrialWavefunctionParams(\n",
+    "    'pp_plus_test',\n",
+    "    hamiltonian_params=pyscf_params,\n",
+    "    heuristic_layers=tuple(),\n",
+    "    do_pp=True,\n",
+    "    restricted=False,\n",
+    "    initial_orbital_rotation=None,\n",
+    "    initial_two_body_qchem_amplitudes=np.asarray([0.3, 0.4]),\n",
+    "    do_optimization=True,\n",
+    "    use_fast_gradients=True\n",
+    ")\n",
+    "\n",
+    "trial_wf = TrialWavefunctionData.build_pp_plus_trial_from_dependencies(\n",
+    "    pp_params, dependencies={pyscf_params: pyscf_hamiltonian}, do_print=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Trial wavefunction energy: {trial_wf.ansatz_energy}\")\n",
+    "print(f\"Exact ground state energy: {trial_wf.fci_energy}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.qcqmc.blueprint import BlueprintParamsTrialWf, BlueprintData\n",
```

Comment:
Should there be a description for this cell?  There's a bunch of cells in a row here with no description besides the short one at the top of the section.

Reply:
Added some description.